src/app/redux/ducks/busy/reducers.js

```diff
@@ -1,12 +1,18 @@
+function getMeta( meta ) {
```

Comment:
very good idea with creating these helper functions but please name them so that it makes more sense when reading the reducer function, ex: `actionShouldBlock`